internal/store/driver.go

```diff
@@ -946,8 +946,7 @@ func (d *Driver) calculateHTTPSEdgesFromGateway(edgeMap map[string]ingressv1alph
 								}
 
 								refName := string(backendref.Name)
-								//refNamespace := string(*backendref.Namespace)
-								serviceUID, servicePort, err := d.getEdgeBackendRef(backendref.BackendRef, gtw.Namespace)
+								serviceUID, servicePort, err := d.getEdgeBackendRef(backendref.BackendRef, httproute.Namespace)
```

Comment:
I'd think it would be the best if we show a different kind of error here. Right now it will render as "not found", but we should probably make different namespaces "not supported" for now.